toolkit/README.md

```diff
@@ -0,0 +1,67 @@
+- [Write Codeblocks](#write-codeblocks)
+  - [Usage](#usage)
+- [Check CLI Output](#check-cli-output)
+  - [Usage](#usage-1)
+
+# Write Codeblocks
+
+The `update_spin_cli_reference.py` script will fetch the current Spin CLI reference document from the web and write a fresh copy of the file (with the new version of code blocks added) to your local disk. This new markdown is the start of a PR that contains CLI commands for the new version of Spin.
+
+## Usage
+
+Edit the version numbers in the file i.e. `2.4`, `2.5` etc. and run the script:
+
+```bash
+python3 update_spin_cli_reference.py
```

Comment:
Possible to update the script such that the versions can be passed in as env vars or args?  (To avoid needing to edit the hard-coded versions in the script itself).